partner_sale_pivot/views/res_partner_views.xml

```diff
@@ -4,7 +4,6 @@
         <field name="name">res.partner.form.sales_by_product</field>
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="sale.res_partner_view_buttons" />
-        <field name="groups_id" eval="[(4, ref('sales_team.group_sale_salesman'))]" />
```

Comment:
Put the group on the button itself.

Reply:
Hey @pedrobaeza ,

if the group already exists in the button.